tests/src/rules/no-empty-named-blocks.js

```diff
@@ -67,7 +78,18 @@ ruleTester.run('no-empty-named-blocks', rule, {
     test({
       code: `import typeof {} from 'mod';`,
       parser: parsers.BABEL_OLD,
-      errors: ['Unexpected empty named import block'],
+      errors: [{
+        suggestions: [
+          {
+            desc: 'Remove unused import',
+            output: '',
+          },
+          {
+            desc: 'Remove empty import block',
+            output: `import  'mod';`,
```

Comment:
any reason this has to keep the extra spaces? i'm not sure if there's another eslint rule that would cover it

Reply:
I thought other rules would take care of it. Just fixed it to remove the extra space.